### DISCUSSION TITLE
Slack Emoji Toolkit

### DISCUSSION THREAD

**Discussion by wilhelm-murdoch** [bashly-framework/bashly#262] [Show and tell]
I had this stupid thought the other day about packing all my custom Slack emojis and taking them with me to a new gig. Next thing I know I'm obsessing over the network panel in my browser's dev console figuring out how Slack's API works. I then went and built this stupid little tool.

Bashly took care of all the grunt work I would usually avoid putting up with and helped me immensely with throwing this together. Thanks heaps for all the good work you're doing with this literal gem! 😗👌🏻 

https://github.com/wilhelm-murdoch/slack-emoji-toolkit

**Comment by DannyBen** [comment]
That is such a cool message, thanks for sharing!
I took a quick look at your `bashly.yml` - looks impressive, I will look at it more thoroughly later - I am curious to see how other people are using it.

**Comment by DannyBen** [comment]
Can I add this to the [Real World Examples](https://bashly.dannyb.co/examples/#real-world-examples) section in the documentation?

**Comment by wilhelm-murdoch** [comment]
Absolutely!

**Reply by DannyBen** [reply]
[Done](https://bashly.dannyb.co/examples/#real-world-examples)
